<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadResultantGUI</class>
 <widget class="QWidget" name="LoadResultantGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Forze</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="dataFromFileXMLPushButton">
        <property name="text">
         <string>-&gt; leggi da file XML</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="dataFromFileTXTPushButton">
        <property name="text">
         <string>-&gt; leggi da file TXT</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QTableView" name="tableViewF"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="addFPushButton">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="delFPushButton">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="dataToFileXMLPushButton">
        <property name="text">
         <string>&lt;- scrivi su file XML</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="addDataCheckBox">
        <property name="text">
         <string>Aggiungi</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="PGroupBox">
     <property name="title">
      <string>Punto applicazione</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QLineEditValue" name="xPLineEdit"/>
      </item>
      <item row="0" column="5">
       <widget class="QLineEditValue" name="yPLineEdit"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabelRichName" name="yPLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;y&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabelUnitMeasure" name="zPUMLabel">
        <property name="text">
         <string>[m]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabelRichName" name="xPLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;x&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabelUnitMeasure" name="xPUMLabel">
        <property name="text">
         <string>[m]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabelUnitMeasure" name="yPUMLabel">
        <property name="text">
         <string>[m]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLineEditValue" name="zPLineEdit"/>
      </item>
      <item row="0" column="6">
       <widget class="QLabelRichName" name="zPLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;z&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="ResultantGroupBox">
     <property name="title">
      <string>Risultante</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QLabelUnitMeasure" name="MxUMLabel">
        <property name="text">
         <string>[N m]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabelUnitMeasure" name="FxUMLabel">
        <property name="text">
         <string>[N]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEditValue" name="FxLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabelRichName" name="FxLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;F&lt;span style=&quot; vertical-align:sub;&quot;&gt;x&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEditValue" name="MyLineEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabelRichName" name="MyLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;M&lt;span style=&quot; vertical-align:sub;&quot;&gt;y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEditValue" name="FyLineEdit"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabelRichName" name="FyLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;F&lt;span style=&quot; vertical-align:sub;&quot;&gt;y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLineEditValue" name="MzLineEdit"/>
      </item>
      <item row="1" column="6">
       <widget class="QLabelRichName" name="MzLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;M&lt;span style=&quot; vertical-align:sub;&quot;&gt;z&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLineEditValue" name="FzLineEdit"/>
      </item>
      <item row="0" column="6">
       <widget class="QLabelRichName" name="FzLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;F&lt;span style=&quot; vertical-align:sub;&quot;&gt;z&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEditValue" name="MxLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabelRichName" name="MxLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;M&lt;span style=&quot; vertical-align:sub;&quot;&gt;x&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabelUnitMeasure" name="FyUMLabel">
        <property name="text">
         <string>[N]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabelUnitMeasure" name="FzUMLabel">
        <property name="text">
         <string>[N]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLabelUnitMeasure" name="MzUMLabel">
        <property name="text">
         <string>[N m]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabelUnitMeasure" name="MyUMLabel">
        <property name="text">
         <string>[N m]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelRichName</class>
   <extends>QLabel</extends>
   <header>qlabelrichname.h</header>
  </customwidget>
  <customwidget>
   <class>QLabelUnitMeasure</class>
   <extends>QLabel</extends>
   <header>qlabelunitmeasure.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditValue</class>
   <extends>QLineEdit</extends>
   <header>qlineeditvalue.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
